common/persistence/src/main/java/app/coronawarn/server/common/persistence/service/TraceTimeIntervalWarningService.java

```diff
@@ -32,10 +39,13 @@ public int saveCheckinData(List<CheckIn> checkins) {
     int numberOfInsertedTraceWarnings = 0;
 
     for (CheckIn checkin : checkins) {
-      boolean traceWarningInsertedSuccessfully = traceTimeIntervalWarningRepo.saveDoNothingOnConflict(
-          checkin.getLocationId().toByteArray(),
-          checkin.getStartIntervalNumber(), checkin.getEndIntervalNumber(),
-          checkin.getTransmissionRiskLevel());
+      boolean traceWarningInsertedSuccessfully = traceTimeIntervalWarningRepo
+          .saveDoNothingOnConflict(checkin.getLocationId().toByteArray(),
+              checkin.getStartIntervalNumber(),
+              checkin.getEndIntervalNumber() - checkin.getStartIntervalNumber(),
```

Comment:
We did this based on the technical spec, period being calculated and store as startIntervalNumber - endIntervalNumber

Reply:
@pithumke 

we receive the `endIntervalNumber`, but have to publish the `period`. In that case, it might makes sense to store the `period` right away, right? (makes also the fake check in generation a bit easier)